src/main/frontend/handler/graph.cljs

```diff
@@ -95,12 +95,17 @@
             pages-after-journal-filter (if-not journal?
                                          (remove :block/journal? full-pages)
                                          full-pages)
+
+           pages-after-exclude-filter (cond->> pages-after-journal-filter
+                                        (not excluded-pages?)
+                                        (remove (fn [p] (=  true (:exclude-in-graph (:block/properties p))))))
```

Comment:
Also, please add this property key to `hidden-built-in-properties` since a page of this property key will be auto-created if not.

Reply:
> Does :exclude-in-graph-view feel more specific since graph means more like file database rather than graph view in Logseq?

Agreed it is more specific. Please also use "from" instead of "in" as it is the more common preposition e.g. `exclude-from-graph-view`

Reply:
> Also, please add this property key to hidden-built-in-properties since a page of this property key will be auto-created if not.

Since this property is editable, the `editable-built-in-properties` var needs to be updated

Reply:
> Also, please add this property key to `hidden-built-in-properties` since a page of this property key will be auto-created if not.
> Since this property is editable, the `editable-built-in-properties` var needs to be updated

I'll work on it

Reply:
> can the is-node property be good?
it cannot be misunderstood (the nodes are only in the global graph, am I wrong?) and above all it is a shorter name

I'd prefer we have a longer, descriptive name so there is no confusion for the user to its purpose. Logseq property names also prevent anyone from using that name for their personal properties. I could see `is-node` being used for a personal property. The next release of logseq has property autocompletion so longer names shouldn't be a problem :)

Reply:
@llcc 
> Also, please add this property key to `hidden-built-in-properties` since a page of this property key will be auto-created if not.

If i'll add :exclude-from-graph-view property in hidden-built-in-properties this property is hide automatically (the value is saved) but in query table result this property will not show

Reply:
@8N9KT No worries. He meant `editable-built-in-properties` so looks like you've updated this correctly